<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Name</string>
  </property>
  <widget class="QWidget" name="centralWidgetMain">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Export"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContent"/>
    <addaction name="actionOnline"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionWidgets_Explorer"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCholor_Scheme_Generator"/>
    <addaction name="actionColor_Scheme_From_Image"/>
    <addaction name="separator"/>
    <addaction name="actionLive_Preview"/>
    <addaction name="separator"/>
    <addaction name="actionCpp"/>
    <addaction name="actionPython"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Widgets Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="4" column="0">
      <widget class="QGroupBox" name="grpboxHVSpacing">
       <property name="title">
        <string>  Spacing</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Horizontal</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sldr_spacing_horizontal">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Vertical</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sldr_spacing_vertical">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="grpboxMargins">
       <property name="title">
        <string>  Margin</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sldr_margin_left">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="sldr_margin_top">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="sldr_margin_right">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="sldr_margin_bottom">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeView" name="treeViewFiles">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frameUiFiles">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnRemoveUiFile">
            <property name="toolTip">
             <string>Remove ui file</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/trash-alt.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/trash-alt.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="btnAddUiFile">
            <property name="toolTip">
             <string>Add ui files</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/plus-circle.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/plus-circle.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTreeView" name="treeViewAppWidgets">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="grpboxSpacing">
       <property name="title">
        <string>  Spacing</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="sldr_spacing">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="action_Export"/>
   <addaction name="separator"/>
   <addaction name="actionWidgets_Explorer"/>
   <addaction name="separator"/>
   <addaction name="actionCholor_Scheme_Generator"/>
   <addaction name="actionColor_Scheme_From_Image"/>
   <addaction name="separator"/>
   <addaction name="actionLive_Preview"/>
   <addaction name="separator"/>
   <addaction name="actionCpp"/>
   <addaction name="actionPython"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-medical.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-medical.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New ...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-alt.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-alt.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-download.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-download.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-download-as.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-download-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>Content</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-export3.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/file-export3.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Export ...</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionWidgets_Explorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/columns.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/columns.svg</iconset>
   </property>
   <property name="text">
    <string>Widgets Explorer</string>
   </property>
  </action>
  <action name="actionCholor_Scheme_Generator">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/magic.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/magic.svg</iconset>
   </property>
   <property name="text">
    <string>Cholor Scheme Generator ...</string>
   </property>
  </action>
  <action name="actionColor_Scheme_From_Image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/camera-retro.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/camera-retro.svg</iconset>
   </property>
   <property name="text">
    <string>Color Scheme From Image ...</string>
   </property>
  </action>
  <action name="actionLive_Preview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/sync-alt.svg</normaloff>:/icons/resources/icons/fontawesome-free-5.10.0-desktop/svgs/solid/sync-alt.svg</iconset>
   </property>
   <property name="text">
    <string>Live Preview</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionOnline">
   <property name="text">
    <string>Online</string>
   </property>
  </action>
  <action name="actionCpp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/cpp.svg</normaloff>:/icons/cpp.svg</iconset>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
   <property name="toolTip">
    <string>Ouput C++ styling</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/python.svg</normaloff>:/icons/python.svg</iconset>
   </property>
   <property name="text">
    <string>Python</string>
   </property>
   <property name="toolTip">
    <string>Output Python styling</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionWidgets_Explorer</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWidgets_Explorer</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_margin_left</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_margin_top</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_margin_right</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_margin_bottom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_spacing_horizontal</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>762</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>763</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_spacing_vertical</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>782</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>783</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_spacing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>704</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>705</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
